Providers/nxOMSAutomationWorker/MSFT_nxOMSAutomationWorkerResource.h

```diff
@@ -29,9 +29,7 @@ typedef struct _MSFT_nxOMSAutomationWorkerResource /* extends OMI_BaseResource *
     MI_Instance __instance;
     /* OMI_BaseResource properties */
     /* MSFT_nxOMSAutomationWorkerResource properties */
-    /*KEY*/ MI_ConstStringField WorkspaceId;
```

Comment:
Keeping near-in-the-future multi-homing compatibility in mind, I would actually recommend keeping WorkspaceId as a key on the schema mof. You can have more than one key in a MOF.